tests/common/widgets/search-box/options.ts

```diff
@@ -0,0 +1,461 @@
+              autocomplete="off"
+              autocorrect="off"
+              class="ais-SearchBox-input"
+              maxlength="512"
+              placeholder=""
+              spellcheck="false"
+              type="search"
+            />
+            <button
+              class="ais-SearchBox-submit"
+              title="Submit the search query"
+              type="submit"
+            >
+              <svg
+                aria-hidden="true"
+                class="ais-SearchBox-submitIcon"
+                height="10"
+                viewBox="0 0 40 40"
+                width="10"
+              >
+                <path
+                  d="M26.804 29.01c-2.832 2.34-6.465 3.746-10.426 3.746C7.333 32.756 0 25.424 0 16.378 0 7.333 7.333 0 16.378 0c9.046 0 16.378 7.333 16.378 16.378 0 3.96-1.406 7.594-3.746 10.426l10.534 10.534c.607.607.61 1.59-.004 2.202-.61.61-1.597.61-2.202.004L26.804 29.01zm-10.426.627c7.323 0 13.26-5.936 13.26-13.26 0-7.32-5.937-13.257-13.26-13.257C9.056 3.12 3.12 9.056 3.12 16.378c0 7.323 5.936 13.26 13.258 13.26z"
+                />
+              </svg>
+            </button>
+            <button
+              class="ais-SearchBox-reset"
+              hidden=""
+              title="Clear the search query"
+              type="reset"
+            >
+              <svg
+                aria-hidden="true"
+                class="ais-SearchBox-resetIcon"
+                height="10"
+                viewBox="0 0 20 20"
+                width="10"
+              >
+                <path
+                  d="M8.114 10L.944 2.83 0 1.885 1.886 0l.943.943L10 8.113l7.17-7.17.944-.943L20 1.886l-.943.943-7.17 7.17 7.17 7.17.943.944L18.114 20l-.943-.943-7.17-7.17-7.17 7.17-.944.943L0 18.114l.943-.943L8.113 10z"
+                />
+              </svg>
+            </button>
+            <span
+              class="ais-SearchBox-loadingIndicator"
+              hidden=""
+            >
+              <svg
+                aria-hidden="true"
+                aria-label="Results are loading"
+                class="ais-SearchBox-loadingIcon"
+                height="16"
+                stroke="#444"
+                viewBox="0 0 38 38"
+                width="16"
+              >
+                <g
+                  fill="none"
+                  fill-rule="evenodd"
+                >
+                  <g
+                    stroke-width="2"
+                    transform="translate(1 1)"
+                  >
+                    <circle
+                      cx="18"
+                      cy="18"
+                      r="18"
+                      stroke-opacity=".5"
+                    />
+                    <path
+                      d="M36 18c0-9.94-8.06-18-18-18"
+                    >
+                      <animatetransform
+                        attributeName="transform"
+                        dur="1s"
+                        from="0 18 18"
+                        repeatCount="indefinite"
```

Comment:
this snapshot is now quite inconsistent, stroke-opacity in kebab-case, but repeatCount, attributeName in camelCase (also I thought the tag was animateTransform, not animatetransform).

Of course most likely this all works fine in html, but I'd still like to see our html output & tests to use only a single style.

Reply:
To me the inconsistency is initially on React's side (screenshot from getting started): 
<img width="1005" alt="CleanShot 2023-11-02 at 11 52 30@2x" src="https://github.com/algolia/instantsearch/assets/154633/40e39117-ae2c-48c5-acbf-ad306a9dd47e">

I believe nor JS neither Vue's DOM outputs are better in this case, we just need to choose one source of truth and make adjustments for the others.